prow/config/jobs.go

```diff
@@ -448,9 +454,20 @@ func (u *UtilityConfig) Validate() error {
 			}
 		}
 
-		if _, err := url.Parse(cloneURI); err != nil {
+		uri, err := url.Parse(cloneURI)
+		if err != nil {
 			return fmt.Errorf("couldn't parse uri from clone_uri: %v", err)
 		}
+
+		if u.DecorationConfig != nil && u.DecorationConfig.OauthTokenSecret != nil {
+			if len(uri.Scheme) == 0 {
+				return fmt.Errorf("couldn't find a scheme from clone_uri: %s", cloneURI)
```

Comment:
nit: we can get rid of this check and cover everything with the one below

Reply:
addressed